playwright/support/constants.ts

```diff
@@ -0,0 +1,28 @@
+export const SIZE = {
+  EXTRASMALL: "extra-small",
+  SMALL: "small",
+  MEDIUMSMALL: "medium-small",
+  MEDIUM: "medium",
+  MEDIUMLARGE: "medium-large",
+  LARGE: "large",
+  EXTRALARGE: "extra-large",
+} as const;
+
+export const VALIDATION = {
+  ERROR: "rgb(203, 55, 74)",
+  WARNING: "rgb(239, 103, 0)",
+  INFO: "rgb(0, 96, 167)",
+} as const;
+
+export const COLOR = {
+  BLACK: "rgb(0, 0, 0)",
+  RED: "rgb(205, 56, 75)",
+  ORANGE: "rgb(255, 156, 75)",
+  BROWN: "rgb(105, 61, 57)",
+} as const;
+
+export const CHARACTERS = {
+  DIACRITICS: "mp150ú¿¡üßä",
+  SPECIALCHARACTERS: "!@#$%^*()_+-=~[];:.,?{}&\"'<>",
+  STANDARD: "cypress_data",
```

Comment:
**suggestion:** Quite trivial but seeing as we're moving away from cypress, should we change this to `playwright_data` or something else completely that is agnostic of the library we use. Something like `foobar_data` as an example.

Reply:
yeah, we can, but I assume will do that after the refactor, as when we are setting let's say `aria-label` to `STANDARD` then we asserting is as well that it should equal `STANDARD`. So only here will be the change :)